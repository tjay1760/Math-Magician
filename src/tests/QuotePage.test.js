import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import Quote from '../components/CalculatorPage';

describe('Should render as expected', () => {
  test('test render of quote page', () => {
    const { container } = render(
      <MemoryRouter>
        <Quote />
      </MemoryRouter>,
    );
    expect(container).toMatchSnapshot();
  });
});
