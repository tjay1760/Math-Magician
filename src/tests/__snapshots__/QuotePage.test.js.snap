// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render as expected test render of quote page 1`] = `
<div>
  <div
    class="calculator"
  >
    <h2>
      Lets do some math 
    </h2>
    <div
      id="my-calculator"
    >
      <section
        id="output-screen"
      >
        <p>
           
          <output>
             
            0
          </output>
        </p>
      </section>
      <div
        id="inputs"
      >
        <div
          id="input-buttons"
        >
          <button
            class="input"
            id="AC"
            type="button"
          >
            AC
          </button>
          <button
            class="input"
            id="+/-"
            type="button"
          >
            +/-
          </button>
          <button
            class="input"
            id="%"
            type="button"
          >
            %
          </button>
          <button
            class="input"
            id="7"
            type="button"
          >
            7
          </button>
          <button
            class="input "
            id="8"
            type="button"
          >
            8
          </button>
          <button
            class="input"
            id="9"
            type="button"
          >
            9 
          </button>
          <button
            class="input"
            id="4"
            type="button"
          >
            4 
          </button>
          <button
            class="input"
            id="5"
            type="button"
          >
            5 
          </button>
          <button
            class="input"
            id="6"
            type="button"
          >
            6 
          </button>
          <button
            class="input"
            id="1"
            type="button"
          >
            1 
          </button>
          <button
            class="input"
            id="2"
            type="button"
          >
            2 
          </button>
          <button
            class="input"
            id="3"
            type="button"
          >
            3
          </button>
          <button
            class="input fill"
            id="0"
            type="button"
          >
            0
          </button>
          <button
            class="input"
            id="."
            type="button"
          >
            .
          </button>
        </div>
        <div
          id="operation-buttons"
        >
          <button
            class="operations"
            id="รท"
            type="button"
          >
            รท
          </button>
          <button
            class="operations"
            id="x"
            type="button"
          >
            x
          </button>
          <button
            class="operations"
            id="-"
            type="button"
          >
            -
          </button>
          <button
            class="operations"
            id="+"
            type="button"
          >
            +
          </button>
          <button
            class="operations"
            id="="
            type="button"
          >
            =
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
